tests/test_eia.py

```diff
@@ -0,0 +1,645 @@
+import unittest
```

Comment:
@ndavis6, unless I'm overlooking something I think I've addressed all the comments here. Please let me know if you see any other issues, particularly with the set_ba function I added to eia_esod.py. Thanks.